### DISCUSSION TITLE
如何不同目标平台的文件

### DISCUSSION THREAD

**Discussion by dengkunkun** [wh201906/SerialTest#51] [Q&A] [answered]
我已经在linux（wsl2）下编译和运行成功了，运行由很多拖影，感觉是wsl2自身的问题
请问要编译windows平台和安卓平台的目标文件要修改哪里

**Comment by wh201906** [comment]
安装Qt的时候勾选MinGW和Android，然后直接在Qt Creator当中编译即可
![图片](https://github.com/wh201906/SerialTest/assets/62299611/de8611b6-f4da-4810-80be-dd39f7786983)

如果需要通过命令行编译，可参考以下CI脚本
https://github.com/wh201906/SerialTest/blob/master/.github/workflows/build-windows.yml
https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/priv.wh201906.serialtest.yml

**Comment by dengkunkun** [comment, marked as answer]
用qt6.6打开，编译用很多报错，用qt5.9编译是成功的

**Reply by wh201906** [reply]
目前只支持Qt5
